packages/core/src/sync-store/migrations.ts

```diff
@@ -1679,6 +1680,64 @@ GROUP BY fragment_id, chain_id
       await sql`ANALYZE ponder_sync.factory_addresses`.execute(db);
     },
   },
+  "2025_02_26_1_rpc_request_results": {
+    async up(db) {
+      await db.schema
+        .alterTable("rpc_request_results")
+        .dropConstraint("rpc_request_result_primary_key")
+        .execute();
+      await db.schema
+        .alterTable("rpc_request_results")
+        .addColumn("request_hash_temp", "text")
+        .execute();
+      await db
+        .updateTable("rpc_request_results")
+        .set({ request_hash_temp: sql`request_hash` })
+        .execute();
+      await db.schema
+        .alterTable("rpc_request_results")
+        .dropColumn("request_hash")
+        .execute();
+      await db.schema
+        .alterTable("rpc_request_results")
+        .renameColumn("request_hash_temp", "request_hash")
+        .execute();
+      await db.schema
+        .alterTable("rpc_request_results")
+        .addPrimaryKeyConstraint("rpc_request_results_pkey", [
+          "request_hash",
+          "chain_id",
+        ])
+        .execute();
+      await db.schema
+        .alterTable("rpc_request_results")
+        .dropColumn("request")
+        .execute();
+      await db
+        .updateTable("rpc_request_results")
+        .set({ block_number: 0 })
+        .where("block_number", "=", maxUint256)
```

Comment:
What's the story here?

Reply:
We need a way to signify "latest" as a block number in the database. Previously this value was `2**256-1`, but since we migrated the column type from numeric(78,0) to bigint, we need a new number.